README.md

```diff
@@ -188,3 +188,24 @@ Then, just add breakpoints by clicking on the line number of your code.
 9. If you'd like to share the query you built, such as demonstrating how you tested your code, check out [Apollo Explorer's sharing features](https://www.apollographql.com/blog/announcement/platform/save-and-share-your-graphql-operations-in-apollo-explorer/#sharing-a-collection).
 
 </details>
+
+<details>
```

Comment:
let's make a new "troubleshooting" section and move this there. 


I'm wondering if there's a better way to programmatically make the indexes instead of having to manually create the indexes every time you restart the emulators or a new dev joins the team. 🤔 
(consider we'll likely have several `get` requests across the app that will need indexes)

Reply:
@ermish I read about automating the creation of indexes. It looked pretty advanced, but I'd like to have a crack at it in the near future.

Reply:
@theyokohamalife @ermish I have tried to find information about the automated creation of indexes, and I honestly can't find anything I understand. I have also been testing the `getSubmissions` function with different combinations of `fieldToOrder` and `orderDirection`. It seems that when testing with the emulator, we don't need to create indexes like we did before adding the emulator.

Should I still move this to a new "troubleshooting" section? Or remove it altogether?

Reply:
I think this is fine to keep here 👍 
You might want to make the header of your information related to the error someone might get